### PR TITLE
Adds baba yaga

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Adds a new abnormality

- It rewards players for learning the ability called dodging.

## Why It's Good For The Game
Uhhh umm uhh

## Changelog
:cl:
add: Added new things
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/waw/babayaga.dm [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+		Y = pick(potentialmarked)
+		potentialmarked -= Y
+		if(Y.stat == DEAD || Y.is_working)
+			continue
+		marked+=Y
+	if(marked.len <= 0) //Oh no, everyone's dead!
+		jump_cooldown = world.time
+		return
+	var/mob/living/carbon/human/final_target = pick(marked)
+	final_target.apply_status_effect(STATUS_EFFECT_BABAYAGA)
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/waw/babayaga.dm:117]: Definitely adding this to hallucinations

File: code/modules/mob/living/simple_animal/abnormality/waw/babayaga.dm [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+	var/turf/target_turf = get_turf(target)
+	forceMove(target_turf) //look out, someone is rushing you!
+	new /obj/effect/temp_visual/giantwarning(target_turf)
+	SLEEP_CHECK_DEATH(10 SECONDS)
+	animate(src, pixel_z = 0, alpha = 255, time = 10)
+	SLEEP_CHECK_DEATH(10)
+	density = TRUE
+	visible_message("<span class='danger'>[src] drops down from the ceiling!</span>")
+	playsound(get_turf(src), 'sound/abnormalities/babayaga/land.ogg', 100, FALSE, 20)
+	var/obj/effect/temp_visual/decoy/D = new /obj/effect/temp_visual/decoy(get_turf(src), src)
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/waw/babayaga.dm:135]:
```suggestion
	var/obj/effect/temp_visual/decoy/D = new(get_turf(src), src)
```
Implied

File: code/modules/paperwork/records/info/waw.dm [hunk 3]
```diff
@@ -445,3 +445,16 @@
 		"DANGER - When Apex Predator breached, it should be located immediately.")
 	abno_breach_damage_type = "Red"
 	abno_breach_damage_count = "Very High"
+
+//Baba Yaga
+/obj/item/paper/fluff/info/waw/babayaga
+	abno_type = /mob/living/simple_animal/hostile/abnormality/babayaga
+	name = "M-04-166"
+	abno_info = list(
+		"When the work result was Normal, the Qliphoth Counter lowered with a low probability.",
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/waw.dm:454]:
```suggestion
		"When the work result was Normal, the Qliphoth Counter lowered with a normal probability.",
```
Low is 20% according to LC iirc. Low/Normal/High is 20/40/80%

File: code/modules/paperwork/records/info/waw.dm [hunk 4]
```diff
@@ -445,3 +445,16 @@
...
+		"When the work result was Bad, the Qliphoth Counter lowered with a high probability. In addition, the employee was attacked by a group of frozen slaves.",
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/waw.dm:455]:
```suggestion
		"When the work result was Bad, the Qliphoth Counter lowered. In addition, the employee was attacked by a group of frozen slaves.",
```
It always lowers on bad.

File: code/modules/paperwork/records/info/waw.dm [hunk 5]
```diff
@@ -445,3 +445,16 @@
...
+		"When an employee, who had both Fortitude Levels and Temperance Levels below 4, completed the work process, the Qliphoth Counter lowered with a high probability. In addition, the employee was attacked by a group of frozen slaves.",
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/waw.dm:456]:
```suggestion
		"When an employee, who had both Fortitude Levels and Temperance Levels below 4, completed the work process, the Qliphoth Counter lowered. In addition, the employee was attacked by a group of frozen slaves.",
```

File: code/modules/paperwork/records/info/waw.dm [hunk 6]
```diff
@@ -445,3 +445,16 @@
...
+		"When Baba Yaga breached, it caused incredibly powerful shockwaves, causing nearby empoloyees to take extreme RED damage if too close to the source.",
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/waw.dm:457]:
```suggestion
		"When Baba Yaga breached, it caused incredibly powerful shock-waves, causing nearby employees to take extreme RED damage proportional to how close they were to the source.",
```

**Review by LanceSmites328 — DISMISSED** [review]

**Review by Kitsunemitsu — APPROVED** [review]
Tested this personally. It was a good abnormality.

**Pull request merged**